OpenRA.Mods.Common/Traits/Attack/AttackFrontal.cs

```diff
@@ -33,26 +39,34 @@ public AttackFrontal(Actor self, AttackFrontalInfo info)
 			this.info = info;
 		}
 
+		public static bool WithinFacingTolerance(int facing, int desiredFacing, int facingTolerance)
+		{
+			if (facingTolerance == 0 && facing == desiredFacing)
+				return true;
+
+			var delta = Util.NormalizeFacing(desiredFacing - facing);
+			if (delta <= facingTolerance || delta >= 256 - facingTolerance)
```

Comment:
This can simplify to `return delta <= facingTolerance || delta >= 256 - facingTolerance;`